src/components/button/_button.scss

```diff
@@ -119,6 +124,7 @@
     @include ie-lte(8) {
       border-bottom: $govuk-border-width-form-element solid $govuk-button-colour-darken-15;
     }
+    line-height: 1 !important;
```

Comment:
Would this prevent wrapping buttons from working? If the button has a lot of text or is on a small screen?

See http://govuk-static.herokuapp.com/component-guide/button/extreme_text_start_now_button

Reply:
It's actually OK... At line-height: 1 it looks like this

<img width="300" alt="screen shot 2017-12-01 at 13 46 10" src="https://user-images.githubusercontent.com/23356842/33485291-b87f2554-d69d-11e7-9c07-468550a56352.png">

I'm know seeing first hand some of the problems with having spacing on a 5px scale and font-sizes of 19 and 24. it means we have to make up the 1px somewhere. Probably from line-height.

<img width="924" alt="screen shot 2017-12-01 at 13 54 34" src="https://user-images.githubusercontent.com/23356842/33485592-d186ce34-d69e-11e7-8bb7-7511d3842b7f.png">